src/css/night/controls/_brush_canvas.sass

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2013-2015 9elements GmbH
+ *
+ * Released under Attribution-NonCommercial 3.0 Unported
+ * http://creativecommons.org/licenses/by-nc/3.0/
+ *
+ * For commercial use, please contact us at contact@9elements.com
+ */
+
+.imglykit-container .imglykit-canvas-brush-container
+  position: absolute
+  top: 0
+  left: 0
+  width: 100%
+  height: 100%
+  cursor: none
+
+#mycursor
```

Comment:
Please don't pollute the global namespace. Use a class name like `.imglykit-canvas-brush-cursor` instead